pkgs/games/factorio/default.nix

```
@@ -53,11 +53,11 @@ let
     x86_64-linux = let bdist = bdistForArch { inUrl = "linux64"; inTar = "x64"; }; in {
       alpha = {
         stable        = bdist { sha256 = "0b4hbpdcrh5hgip9q5dkmw22p66lcdhnr0kmb0w5dw6yi7fnxxh0"; version = "0.16.51"; withAuth = true; };
```

also got a different sha here.
```suggestion
        stable        = bdist { sha256 = "1g5q19r0ggdnp8272l7s2h9gd4hqpdsxxkq7vhjsp1kl96nq8ays"; version = "0.16.51"; withAuth = true; };
```

---

I did not change this at all, and I don't believe the hash has changed. This builds for me with no change, but only does because I setup API keys. Note that it does actually download the bundle from the website.